--- ./../original/Dollchan_Extension_Tools.es6.user.js	2018-10-13 17:54:57.092878000 +0000
+++ ./../Dollchan_Extension_Tools.es6.user.js	2018-11-12 04:06:21.941733283 +0000
@@ -1,12 +1,11 @@
 // ==UserScript==
-// @name            Dollchan Extension Tools
+// @name            Dollchan Extension Tools (Cirno Edition)
 // @version         18.8.9.0
 // @namespace       http://www.freedollchan.org/scripts/*
 // @author          Sthephan Shinkufag @ FreeDollChan
 // @copyright       © Dollchan Extension Team. See the LICENSE file for license rights and limitations (MIT).
 // @description     Doing some profit for imageboards
 // @icon            https://raw.github.com/SthephanShinkufag/Dollchan-Extension-Tools/master/Icon.png
-// @updateURL       https://raw.github.com/SthephanShinkufag/Dollchan-Extension-Tools/master/Dollchan_Extension_Tools.meta.js
 // @run-at          document-start
 // @grant           GM_getValue
 // @grant           GM_setValue
@@ -18,7 +17,9 @@
 // @grant           GM.deleteValue
 // @grant           GM.xmlHttpRequest
 // @grant           unsafeWindow
-// @include         *
+// @match           https://2ch.hk/*
+// @match           https://iichan.hk/*
+// @match           http://dobrochan.com/*
 // @nocompat        Chrome
 // ==/UserScript==
 
@@ -16770,43 +16771,7 @@
 
 // Checking for Dollchan updates from github
 function checkForUpdates(isManual, lastUpdateTime) {
-	if(!isManual) {
-		if(Date.now() - +lastUpdateTime < [0, 1, 2, 7, 14, 30][Cfg.updDollchan] * 1e3 * 60 * 60 * 24) {
-			return Promise.reject();
-		}
-	}
-	return $ajax(
-		gitRaw + 'src/modules/Wrap.js', { 'Content-Type': 'text/plain' }, false
-	).then(({ responseText }) => {
-		const v = responseText.match(/const version = '([0-9.]+)';/);
-		const remoteVer = v && v[1] ? v[1].split('.') : null;
-		if(remoteVer) {
-			const currentVer = version.split('.');
-			const src = `${ gitRaw }${ nav.isESNext ? 'src/' : '' }Dollchan_Extension_Tools.${
-				nav.isESNext ? 'es6.' : '' }user.js`;
-			saveCfgObj('lastUpd', Date.now());
-			const link = `<a style="color: blue; font-weight: bold;" href="${ src }">`;
-			const chLogLink = `<a target="_blank" href="${ gitWiki }${
-				lang === 1 ? 'versions-en' : 'versions' }">\r\n${ Lng.changeLog[lang] }<a>`;
-			for(let i = 0, len = Math.max(currentVer.length, remoteVer.length); i < len; ++i) {
-				if((+remoteVer[i] || 0) > (+currentVer[i] || 0)) {
-					return `${ link }${ Lng.updAvail[lang].replace('%s', v[1]) }</a>${ chLogLink }`;
-				} else if((+remoteVer[i] || 0) < (+currentVer[i] || 0)) {
-					break;
-				}
-			}
-			if(isManual) {
-				const c = responseText.match(/const commit = '([0-9abcdef]+)';/)[1];
-				const vc = version + '.' + c;
-				return c === commit ? Lng.haveLatestCommit[lang].replace('%s', vc) :
-					`${ Lng.haveLatestStable[lang].replace('%s', version) }\r\n${
-						Lng.newCommitsAvail[lang].replace('%s', `${ link }${ vc }</a>${ chLogLink }`) }`;
-			}
-		}
-		return Promise.reject();
-	}, () => !isManual ?
-		Promise.reject() : `<div style="color: red; font-weigth: bold;">${ Lng.noConnect[lang] }</div>`
-	);
+
 }
 
 function initPage() {
